frontend/src/main/utils/sortHelper.js

```diff
@@ -0,0 +1,55 @@
+
+// compareValues comes from this article
+//   https://www.sitepoint.com/sort-an-array-of-objects-in-javascript/
+// It is used to sort arrays of objects, like this:
+// For
+//  const singers = [
+//     { name: 'Steven Tyler', band: 'Aerosmith', born: 1948 },
+//     { name: 'Karen Carpenter', band: 'The Carpenters', born: 1950 },
+//     { name: 'Kurt Cobain', band: 'Nirvana', born: 1967 },
+//     { name: 'Stevie Nicks', band: 'Fleetwood Mac', born: 1948 },
+//   ];
+//
+// Use: 
+// // array is sorted by band, in ascending order by default
+// singers.sort(compareValues('band'));
+
+// // array is sorted by band in descending order
+// singers.sort(compareValues('band', 'desc'));
+
+// // array is sorted by name in ascending order
+// singers.sort(compareValues('name'));
+
+// // array is sorted by date if birth in descending order
+// singers.sort(compareValues('born', 'desc'));
+
+// Stryker disable next-line StringLiteral: The "" mutant results in equivalent mutant due to it defaulting to ascending sort if order is not "desc"
+export function compareValues(key, order = 'asc') {
+    return function innerSort(a, b) {
+      // Logical Operator: Requiring only one object to have the property causes comparisons to undefined to be equivalent (> or < to undefined evaluates to false which will return 0)
+      // Conditional Expression: Changing to false could have both properties be undefined but < > comparisons to undefeind are equivalent to false, returning 0 anyway
+      // BlockStatement: Removing "Return 0" reults in equivalent mutant since one or both elements will be undefined and < > comparisons will evaluate to false
+      // Stryker disable next-line LogicalOperator, ConditionalExpression, BlockStatement: Each mutant results in equivalent code (reasons in above code comment)
```

Comment:
same here

Reply:
Again, line 32 is needed so that mutation testing passes.  The comments on lines 29,30,31 explain what is going on with the mutation testing exception.  So in this case, I think it's good to keep them.